### ISSUE TITLE
FilePermissions has a typo in the default unix value

### ISSUE THREAD

**Issue by ben-manes** [gradle/gradle#25830]
### Issue type

Typo

### Problem description

> FILE: read & write for OWNER, read for GROUP, read for OTHER (0644, r-wr--r--)

This should be `rw-r--r--` as it is `rwx` segments.

### Context (optional)

This typo is listed in the [guide](https://docs.gradle.org/8.3-rc-1/userguide/working_with_files.html#sec:setting_file_permissions) and [javadoc](https://docs.gradle.org/8.3-rc-1/javadoc/org/gradle/api/file/FilePermissions.html).

### Page with the problem

https://docs.gradle.org/8.3-rc-1/userguide/working_with_files.html#sec:setting_file_permissions

**Comment by ov7a** [comment]
Thank you for your interest in Gradle!

This is a valid documentation issue that we will address.

***

This issue is a good choice for first-time contributors to Gradle, it is actionable and ready for contribution.

See [CONTRIBUTING.md](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md) for more information.

**Comment by pratik2315** [comment]
Hey! I would like to work on this issue :)

**Comment by ov7a** [comment]
@pratik2315 Ok, thank you!

**Comment by ben-manes** [comment]
As stated twice before, please also fix the JavaDoc. The fix only changed the user manual.

https://github.com/gradle/gradle/blob/6b59be08e636a18535ae4747c3b02a65339055a0/subprojects/core-api/src/main/java/org/gradle/api/file/FilePermissions.java#L39

**Comment by ov7a** [comment]
@pratik2315 Can you take care of this?

**Comment by ben-manes** [comment]
fyi, 8.3 did not cherry-pick the user guide typo fix nor is the javadoc fixed.

**Comment by ov7a** [comment]
@ben-manes Yep, it would be included in 8.4 It's not severe enough to make the 8.3.1 release. 
I've fixed the Javadoc typo.
